account_move_default_journal/models/account_config_settings.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2021 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    account_move_default_general_journal_id = fields.Many2one(
```

Comment:
This will not really work in a multi-company situation as journals are always linked to a specific company.

Reply:
Have a look at option two here for company specific settings.  Note that set_values and get_values are not really needed any more if you use the config_parameter key in the field definition (see option 3 in the link): 
https://gist.github.com/ryanc-me/a0e35fb6741d73c39f9e3968eaa485ee